fs/shm/shm_unlink.c

```diff
@@ -155,8 +155,8 @@ int shm_unlink(FAR const char *name)
   if (ret < 0)
     {
       set_errno(-ret);
-      return -1;
+      return ERROR;
     }
 
-  return 0;
+  return ret;
```

Comment:
Here I will fix. My bad.

Reply:
I mean it should be `OK` and not `ret`. Just to filter out positive values.

Reply:
Ok, OK is 0 so the functionality is according to the spec. Maybe the new style looks better to other people :)

Reply:
I just looked into code again and all is fine. I put `ret` here just because `file_shm_unlink` has `DEBUGASSERT(ret == OK);`. Let's keep it as is